com.mikrotik.WinBox.yml

```diff
@@ -0,0 +1,61 @@
+id: com.mikrotik.WinBox
+runtime: org.freedesktop.Platform
+runtime-version: "24.08"
+sdk: org.freedesktop.Sdk
+command: winbox
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --share=network
+
+modules:
+  - name: WinBox
+    buildsystem: simple
+    build-commands:
+    - install -Dm755 apply_extra /app/bin/apply_extra
+    - install -Dm755 winbox /app/bin/winbox
+    - install -Dm644 winbox.desktop /app/share/applications/${FLATPAK_ID}.desktop
+    - desktop-file-edit --set-icon=${FLATPAK_ID} /app/share/applications/${FLATPAK_ID}.desktop
```

Comment:
since you provide the .desktop file then there is no reason to have to edit it. It should be right.